problem_builder/templates/html/dashboard.html

```diff
@@ -42,16 +42,30 @@
           {% for mcq in block.mcqs %}
             <tr>
               <th class="desc">{{ mcq.display_name }}</th>
-              <td class="value" {% if mcq.color %}style="border-right-color: {{mcq.color}};"{% endif %}>
-                {% if mcq.value %}{{ mcq.value }}{% endif %}
+              <td class="value"
+                  {% if mcq.color %} style="border-right-color: {{mcq.color}};"{% endif %}
+                  {% if not show_numbers %}
+                    {% if mcq.value %} aria-label="Score: {{mcq.value}}" {% else %} aria-label="{% trans 'No value yet' %}" {%endif%}
```

Comment:
@e-kolpakov I've never seen anyone try to use ```aria-label``` on a ```td``` element, and I don't blame you for trying since the [spec says it is allowed](http://www.w3.org/TR/html-aria/#document-conformance-requirements-for-use-of-aria-attributes-in-html).  However, I wanted to perform a quick implementation test for [using aria-label on a td element](http://sadecki.com/sandbox/test/aria-label-on-td.html) and I found that only Voiceover will read the ```aria-label```, and only if there is an explicit value in the ```td``` element, and it reads *both*.  If the cell is empty, it ignores the ```aria-label```.  Testing in NVDA and JAWS revealed that *only* explicit ```td``` values are read. ```aria-labels``` are ignored.  

I recommend using off-screen text (class=sr) inside the table cells in this use case.